test/extended/clusterup.sh

```diff
@@ -158,6 +158,21 @@ function os::test::extended::clusterup::hostdirs () {
     BASE_DIR="${base_dir}" os::test::extended::clusterup::internal::hostdirs
 }
 
+# Tests bringing up the service catalog and provisioning a template
+function os::test::extended::clusterup::service_catalog() {
```

Comment:
stems from an IRC comment from @bparees ... mentioned that no ext testing of `oc cluster up --service-catalog` existed today, and he was thinking I might be able to add some 

this is my foray into that 

Is it WIP ? .... not sure how much more or less it is compared to the other aspects of the PR, and the pending upstream SC/SB dependencies we have.

I guess the first question is whether this was the type of ext testing @bparees was referring to.